stale-blocks.csv

```diff
@@ -1,3 +1,4 @@
+height,hash,header
```

Comment:
This seams to break the sanity check in CI https://github.com/bitcoin-data/stale-blocks/pull/6/checks#step:3:10

Reply:
I think something like

```python
next(reader, None)  # skip the headers
```
should work